packages/common/src/HardhatConfig.js

```diff
@@ -44,7 +44,15 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
     },
     ovm: { solcVersion: "0.8.4-broken_alpha" },
     networks: {
-      hardhat: { gas: 11500000, blockGasLimit: 11500000, allowUnlimitedContractSize: false, timeout: 1800000 },
+      hardhat: {
+        hardfork: "london",
+        gasPrice: "auto",
```

Comment:
This is pretty cool -- might be nice to use this on other networks too so we don't need to use our gas estimator in scripts!